server/models/Collective.js

```diff
@@ -225,6 +225,8 @@ export default function(Sequelize, DataTypes) {
 
       address: DataTypes.STRING,
 
+      countryISO: DataTypes.STRING,
```

Comment:
Would be nice to have validations for `len: 2` and `isAlpha: true` on this one (or with a regex or any other trick). See http://docs.sequelizejs.com/manual/tutorial/models-definition.html#validations